lua/octo/commands.lua

```diff
@@ -62,6 +62,17 @@ function M.setup()
       close = function()
         M.change_state "CLOSED"
       end,
+      develop = function(repo, ...)
+        local bufnr = vim.api.nvim_get_current_buf()
+        local buffer = octo_buffers[bufnr]
+
+        if buffer and buffer.kind and buffer.kind == "issue" then
+          utils.develop_issue(buffer.node.number)
+        else
+          local opts = M.process_varargs(repo, ...)
+          picker.issues(opts, true)
```

Comment:
Do you expect to pass any of the arguments handled by varargs to `Octo issue develop`? If so, this should be ok

Reply:
Do not expect to pass any args here. Was using the functions that already exist in order to get the current repo (when repo is not passed). But could just duplicate the logic instead. 

This works as is but could cause some confusion if digging into this.

What are your thoughts?

Reply:
I think we can pass `repo` and `develop` as part of the opts so there should not be any need for handling varargs or passing develop as a separate argument.

Reply:
Are you suggesting a change then?

Reply:
The issue with passing `repo` as an argument is that the location of that repo locally is not known. So, only the current repo is truely supported.

Reply:
I can create an issue to clean up this in the future